docs/backend-requests/making/custom-session-token.mdx

```diff
@@ -40,11 +40,11 @@ This guide will show you how to customize a session token to include additional
     export default async function Page() {
       const { sessionClaims } = await auth()
 
-      const firstName = sessionClaims?.fullName
+      const fullName = sessionClaims?.fullName
```

Comment:
I've updated to `fullName` since that's what's already present in existing screenshots.